cmd/autoscaler/main.go

```diff
@@ -201,8 +164,81 @@ func main() {
 	case <-egCh:
 	case <-stopCh:
 	}
+}
+
+func setupLogger() (*zap.SugaredLogger, zap.AtomicLevel) {
+	loggingConfigMap, err := configmap.Load("/etc/config-logging")
+	if err != nil {
+		log.Fatalf("Error loading logging configuration: %v", err)
+	}
+	loggingConfig, err := logging.NewConfigFromMap(loggingConfigMap)
+	if err != nil {
+		log.Fatalf("Error parsing logging configuration: %v", err)
+	}
+	return logging.NewLoggerFromConfig(loggingConfig, component)
+}
+
+func setupChannels() (<-chan struct{}, chan *autoscaler.StatMessage) {
+	stopCh := signals.SetupSignalHandler()
+	statsCh := make(chan *autoscaler.StatMessage, statsBufferLen)
+	return stopCh, statsCh
+}
```

Comment:
Let's inline these bits please. The channels have nothing to do with each other and I think these two lines won't harm in the main function? I feel like it's clearer without factoring out.

Reply:
You're right, reverted.